internal/binutils/disasm.go

```diff
@@ -37,6 +37,7 @@ func findSymbols(syms []byte, file string, r *regexp.Regexp, address uint64) ([]
 	var symbols []*plugin.Sym
 	names, start := []string{}, uint64(0)
 	buf := bytes.NewBuffer(syms)
+	symAddr := uint64(0)
```

Comment:
I think this symAddr will get shadowed by the assignment in the loop header, so it's probably not what the code after the loop expects.

Reply:
Well spotted!